website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { VFC, useState } from 'react';
+import close from 'assets/images/close.svg';
+import refresh from 'assets/images/refresh2.svg';
+import selected from 'assets/images/radio-selected.svg';
+import deselected from 'assets/images/radio-deselected.svg';
+import copy from 'assets/images/copy.svg';
+import { copyToClipboard, isNodeAddressValid } from 'helpers';
+import { Trash2 } from 'react-feather';
+import { nodeApi } from 'api/initApi';
+import { useAlert } from 'react-alert';
+import { useDispatch } from 'react-redux';
+import { resetApiReady } from 'store/actions/actions';
+import { useHistory, useLocation } from 'react-router-dom';
+import { NODE_ADRESS_URL_PARAM } from 'consts';
+import * as init from './init';
+import './Sidebar.scss';
+
+type Props = {
+  closeSidebar: () => void;
+};
+
+const Sidebar: VFC<Props> = ({ closeSidebar }) => {
```

Comment:
I think Sidebar is not best name for this kind of component, you could have a look at [mui](https://mui.com/components/drawers/) or [bootstrap](https://getbootstrap.com/docs/5.0/components/offcanvas/).

Reply:
Sidebar is chosen by a collective decision.